SPEC.md

```diff
@@ -2640,6 +2641,15 @@ This function will return `false` if the argument is an unset optional value. It
 - This function returns the basename of a file path passed to it: `basename("/path/to/file.txt")` returns `"file.txt"`.
 - Also supports an optional parameter, suffix to remove: `basename("/path/to/file.txt", ".txt")` returns `"file"`.
 
+## Int floor(Float), Int ceil(Float) and Int round(Float)
+
+:pig2: [Supported in Cromwell 28](https://github.com/broadinstitute/cromwell#wdl-support) :white_check_mark:
```

Comment:
Hmm, this is actually copypasta. Whether `Cromwell supports` belongs in a spec is another issue but since it's here at all. I think that saying which version adds value.